pkg/cmd/server/server.go

```diff
@@ -78,10 +79,10 @@ const (
 )
 
 type serverConfig struct {
-	pluginDir, metricsAddress, defaultBackupLocation string
-	backupSyncPeriod, podVolumeOperationTimeout      time.Duration
-	restoreResourcePriorities                        []string
-	restoreOnly                                      bool
+	pluginDir, metricsAddress, defaultBackupLocation              string
+	backupSyncPeriod, podVolumeOperationTimeout, namespaceTimeout time.Duration
```

Comment:
since we're using this value for PV waiting as well, maybe we want to call it something more generic, like `terminatingResourceTimeout` or sthg along those lines